src/test/java/org/fundacionjala/coding/maria/CifradoCesarTest.java

```diff
@@ -0,0 +1,164 @@
+package org.fundacionjala.coding.maria;
+
+import org.fundacionjala.coding.maria.cifrado.CifradoCesar;
+import org.fundacionjala.coding.maria.cifrado.CifradoVigenere;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * class cifrado cesar test.
+ */
+public class CifradoCesarTest {
+    private CifradoCesar cesar;
+    private CifradoVigenere vigenere;
+    private static final Object THREE = 3;
+    private static final Object NEGONE = -1;
+    private static final Object ONE = 1;
+    private static final String CASE1 = "HOLA JALASOFT";
+    private static final String LIMON = "LIMON";
+    private static final Object CASE2 = null;
+    /**
+     * initialize cifrado cesar.
+     */
+    @Before
+    public void initialize() {
+        cesar = new CifradoCesar();
+        vigenere = new CifradoVigenere();
```

Comment:
This is wrong
because you are not using the two instances in all the tests

Consider splitting in two unit test class